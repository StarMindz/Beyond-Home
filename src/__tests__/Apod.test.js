import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import APOD from '../components/pages/Apod';

describe('APOD page', () => {
  test('Show Correct render of APOD', () => {
    const apod = render(
      <React.StrictMode>
        <Provider store={store}>
          <Router>
            <APOD />
          </Router>
        </Provider>
      </React.StrictMode>,
    );
    expect(apod).toMatchSnapshot();
  });
});
