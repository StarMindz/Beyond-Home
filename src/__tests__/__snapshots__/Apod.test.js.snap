// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APOD page Show Correct render of APOD 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="apodContainer"
      >
        <h1
          class="apodHeading"
        >
          Astronomy Picture of the day
        </h1>
        <h2
          class="apodDate"
        />
        <h2
          class="apodTitle"
        />
        <img
          alt="Astronomy of the Day"
          class="apodImage"
        />
        <p
          class="apodDescription"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="apodContainer"
    >
      <h1
        class="apodHeading"
      >
        Astronomy Picture of the day
      </h1>
      <h2
        class="apodDate"
      />
      <h2
        class="apodTitle"
      />
      <img
        alt="Astronomy of the Day"
        class="apodImage"
      />
      <p
        class="apodDescription"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
